PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -110,32 +112,38 @@ class SearchViewModel: ObservableObject {
     func clear() {
         searchResults = []
         subscriptions = []
+        savesLocalSearch.clear()
+        savesOnlineSearch.clear()
+        archiveOnlineSearch.clear()
+        allOnlineSearch.clear()
```

Comment:
Instead of clearing, let's instantiate new search instances. This prevents the likelihood of us accidentally perpetuating state from one search into the next.